executor/load_data.go

```diff
@@ -505,67 +461,32 @@ loop:
 // getLine returns a line, curData, the next data start index and a bool value.
```

Comment:
Comments need to be updated.

Reply:
comments here no need to change, `getLine` keeps its original logic